app/controllers/declaration/ItemTypeController.scala

```diff
@@ -196,33 +199,37 @@ class ItemTypeController @Inject()(
           nationalAdditionalCodes = removeElement(itemTypeCached.nationalAdditionalCodes, label.index)
         )
     }
-    updateExportsCache(itemId, itemTypeUpdated).flatMap { _ =>
-      val itemTypeInput: ItemType = ItemType.form().bindFromRequest().value.getOrElse(ItemType.empty)
-      refreshPage(itemId, itemTypeInput, hasFiscalReferences)
+    updateExportsCache(itemId, itemTypeUpdated).map {
+      case Some(model) =>
+        val itemTypeInput: ItemType = ItemType.form().bindFromRequest().value.getOrElse(ItemType.empty)
+        refreshPage(itemId, itemTypeInput, model)
+      case None =>
+        BadRequest("Model not updated")
```

Comment:
Similar here, lets just redirect somewhere sensible if this happens rather than risking a bad user experience